src/lib/shared/api/adapters.ts

```diff
@@ -69,3 +82,7 @@ export const transformPodcastRequest = (podcast: SBPodcast): Podcast | null => {
       .sort((e1, e2) => e1.episodeNumber - e2.episodeNumber),
   };
 };
+
+export const transformFavourite = (favourite: SBFavourite): Episode['id'] => {
```

Comment:
I don't see this function being used anywhere. I also don't feel `(f) => f.episode_id` is worth defining a dedicated function unless it's used more than once. Plus, the name doesn't really convey what it actually does

Reply:
Ah, right, I meant to refactor some of Magomed's new code to use this function. It does convey what the function does — it transforms the Supabase representation of favourites to our representation of favourites (check me out, juggling `u`)